docker-compose.yml

```diff
@@ -1,18 +1,17 @@
-version: '2'
+version: "3"
 services:
   cloudant:
     image: ibmcom/cloudant-developer:latest
```

Comment:
This image is retired, I would suggest we use `ibmcom/couchdb3` instead, see https://hub.docker.com/r/ibmcom/couchdb3.